### PR TITLE
BEASW-17784: Upgrade reports dependencies to the latest version

### PR DESCRIPTION
**Pull request by gmartinez3werfen**
# PR Details

Updates third party dependencies.

## Motivation and Context

Reduce number of vulnerabilities.

Please include relevant motivation and context. List any dependencies that are required for this change.

## Implementation

Update pom.xml file to include latest stable release versions of any dependency.

## How Has This Been Tested

All existing tests has been run with full success. No additional tests has been added as does not include any new feature.

## Types of changes

- [x ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] New release version
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] I have read the **CONTRIBUTING** document
- [x] My code follows the code style of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] There are **component tests** for new/changed code (or a good explanation why this was not possible) and all of
  them passed. I have also added tests to cover my changes
- [x] All **CI builders** have indicated success (Give them the needed time to notice the pull request)
- [x] The **Pull request title** has the type of change ended with a slash (bugfix/feature/...), Jira issue numbers
  separated by spaces (if any), a space, and then a short, but descriptive summary
- [x] **Commit messages** are well formed: [A note about Git commit messages](http://www.tpope.net/node/106)
- [ ] New public packages, classes, and methods are **documented**. (Strongly consider documenting private classes and
  methods)
- [ ] A new branch needs to be created from master to evolve previous versions
- [x] Increase version in pom.xml following [Semantic Versioning](https://semver.org/)
- [ ] Squash merge as a recommended merge strategy in Bitbucket

### CODE REVIEW

**Review by davidberruezo — APPROVED** [review]
I approve these changes

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ To get you started you can simply clone the `java-report-test-utilities` reposit
 
 You need [git][git] to clone the `java-report-test-utilities` repository.
 
-You will need [Java™ SE Development Kit 8][jdk-download] and [Maven][maven].
+You will need [Java™ SE Development Kit 21][jdk-download] and [Maven][maven].
```
Comment (joaquimvila) [inline README.md]: Let's update here the JDK link to point to Eclipse Temurin.

**Review by joaquimvila — APPROVED** [review]
Agree with implementation, only a single documentation update requested. Good job @gmartinez3werfen!

File: README.md (hunk 1 above)
Comment (gmartinez3werfen) [reply README.md]: Done

**Review by QuimNotoTrilla — APPROVED** [review]

**Pull request merged**